src/controllers/admin.js

```diff
@@ -135,11 +135,17 @@ adminController.categories.get = function(req, res, next) {
 		if (err) {
 			return next(err);
 		}
-
-		res.render('admin/manage/category', {
-			category: data.category[0],
-			privileges: data.privileges
-		});
+        
+        plugins.fireHook('filter:admin.categories.get', {req: req, res: res, category: data.category[0], privileges: data.privileges}, function(err, data) {
```

Comment:
We're retrieving one category, so this hook should be` filter:admin.category.get`